README.md

```diff
@@ -13,7 +13,7 @@ We will address these as soon as we can, but there are no specific SLAs.
 
 This application aims to help monitor Pure Storage FlashArrays by providing an "exporter", which means it extracts data from the Purity API and converts it to the OpenMetrics format, which is for instance consumable by Prometheus, or other observability platforms.
 
-The stateless design of the exporter allows for easy configuration management as well as scalability for a whole fleet of Pure Storage arrays. Each time the OpenMetrics client scrapes metrics for a specific system, it should provide the hostname via GET parameter and the API token as Authorization token to this exporter.
+The stateless design of the exporter allows for easy configuration management as well as scalability for a whole fleet of Pure Storage arrays. The design follows almost completely the [multi-target-exporter](https://prometheus.io/docs/guides/multi-target-exporter/) pattern described in the Prometheus documentation, so the tool can be used to scrape multiple FlashArrays from a single instance or just act the front-end for a single FlashArray. Each time the OpenMetrics client scrapes metrics for a specific system, it should provide the target array name or IP address via GET parameter to this exporter.
```

Comment:
We should state that the authorization tokens can still be passed via Bearer Authorization token in the GET request.

Reply:
I edited this so remove the last sentence and promoted the Authentication section to a full header.
This should make the whole thing read cleaner